### DISCUSSION TITLE
How to use FinishResponse to ge the complete result

### DISCUSSION THREAD

**Discussion by dalvirsaini** [betalgo/openai#248] [Q&A]
I am using the following code to get the result. But here I have a result which is trimmed by AI. How I can get the remaining result? 
Following code is used by me to hit AI
        
        var completionResult = await _openAIService.ChatCompletion.CreateCompletion(new ChatCompletionCreateRequest
                {
                    Messages = new List<ChatMessage>
                    {
                      ChatMessage.FromSystem($"You are Event Industry AI Assistant"),
                      ChatMessage.FromUser(prompt)
                    },
                    N = generateContentInput.OutputCount,
                    MaxTokens = maxTokens
                });

                if (completionResult.Successful)
                {
                    var outputs = completionResult.Choices.Select(a => a.Message.Content).ToList();

                    var histories = new List<AITemplateHistory>();
                    foreach (var output in outputs)
                    {

**Comment by firstsaofan** [comment]
If you want AI to reply to your complete content and contact the context, please bring your recent questions and previous AI's answers when sending the request.

**Comment by dalvirsaini** [comment]
I am getting a response like 
{
	"id": "chatcmpl-79RytAFMlSx8pbSqxVGsTRCV8FCDE",
	"object": "chat.completion",
	"created": 1682486867,
	"model": "gpt-3.5-turbo-0301",
	"usage": {
		"prompt_tokens": 1180,
		"completion_tokens": 500,
		"total_tokens": 1680
	},
	"choices": [{
		"message": {
			"role": "assistant",
			"content": "long contents are here"
		},
		"finish_reason": "length",
		"index": 0
	}]
}

Now as per finish_reason I want to complete message of the choice 
what is the way to continue message  here??

**Comment by dm-shlandii** [comment]
Struggling with same question. So what is the way to get the complete response ? 

Tried to take response and add it to Messages collection with .FromAssistant.
